docker/testrunner/entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+cd /black-screen
+
+if [[ ! -e /.firstrun ]] && ( [[ ! -d node_modules ]] || [[ -n $FORCE ]] )
```

Comment:
`firstrun` sounds like it should be present only on the first run, but it's the opposite.